baseapp_payments/views.py

```diff
@@ -56,9 +57,9 @@ def retrieve(self, request, remote_subscription_id=None):
 
     def delete(self, request):
         remote_subscription_id = request.query_params.get("remote_subscription_id")
-        customer = Customer.objects.filter(entity_id=self.request.user.id).first()
+        customer = get_customer(False, request.user)
```

Comment:
We would have a bug here for example, since `remote_customer_id` is False on `get_customer` then it would always try to delete the subscription for `user.profile`, but what if you are trying to delete a subscription for another profile? (assuming you have permission). I think that way it is built now you would never be able to delete an Organization's subscription for example.